benchmarks/hia/Makefile

```
@@ -1,18 +1,43 @@
 bench: build
 	hyperfine --export-csv hia.csv \
```

```suggestion
	hyperfine --export-csv results.csv \
```

I just noticed that the benchmarks for the other systems store the results in a file `results.csv`.